### ISSUE TITLE
Encouraging copying licensed content via Divmagic from Flowbite

### ISSUE THREAD

**Issue by zoltanszogyenyi** [DivMagicCom/DivMagic#7]
Dear @RealBrianH,

I write to you in my capacity as one of the co-founders of [Flowbite](https://flowbite.com/pro/), gravely concerned about an issue that has been brought to our attention via our client support channels. It has come to our notice that DivMagic is actively promoting and facilitating the unauthorized replication and utilization of our encrypted iFrames from the premium version of our service. We must emphasize that such actions constitute a clear violation of our copyrighted material, and any use of our premium content without the requisite license is a blatant infringement of our intellectual property rights.

We urgently request the immediate removal of the infringing website section from your platform, as well as the cessation of any further promotion or facilitation of this unauthorized behavior. It is imperative that you recognize the significant moral and legal implications associated with your actions, which not only undermine the efforts of our contributors and employees but also jeopardize the integrity of the Flowbite community as a whole.

Should you fail to take prompt and decisive action to rectify this situation, we will regrettably be compelled to pursue all available legal remedies to protect our intellectual property rights.

Thank you for your attention to this matter.

Yours sincerely,

Zoltan
Co-founder, Flowbite

<img width="1616" alt="Screenshot 2024-03-19 at 10 49 38" src="https://github.com/DivMagicCom/DivMagic/assets/8052108/92a0ce64-c4bd-4c94-af2c-e9971ddb0b55">

**Comment by RealBrianH** [comment]
Hello @zoltanszogyenyi,

Thank you for your message and concerns.

DivMagic was created to enhance the web development process by leveraging the inherent capabilities of web browsers, including the built-in feature that allows for copying code directly from websites. This functionality is universally accessible and facilitated by a simple "Copy" button present in all modern browsers including Chrome and Firefox. Our tool aims to streamline this existing process for educational and development efficiency, not to bypass or undermine the protections content creators have over their work.

Regarding your specific concern about encrypted iFrames within Flowbite's premium service, it's important to note that DivMagic does not possess any decryption capabilities. Our tool can only access and copy code that is directly visible within iFrames, without any form of decryption. If content is encrypted, it remains inaccessible to DivMagic. We can not decrypt any code or access any web page that is not publicly available. This ensures our adherence to legal standards and respect for copyright protection, highlighting that our platform only facilitates the use of publicly visible and accessible code.

We saw the list of resources on your website (https://flowbite.com/resources/), where some promote functionality similar to DivMagic's. This presents a curiosity regarding the stance on tools with copying capabilities, especially when such tools are highlighted and promoted on your own platform. I mention this not as a critique but to seek clarity on the standards applied within our shared tech community. Why are certain tools that can copy code from Flowbite promoted, while others are scrutinized?

It was never our intention to harm or devalue the work of Flowbite or any other entity. We are going to remove the video section you mentioned.

Additionally, I want to share that our team holds Flowbite in high regard, with all of us being paid developer license holders and a part of Flowbite community. Our admiration for Flowbite was a key reason for its inclusion in our video.

I’ll update the website today and resolve the issue after that.

Best,
Brian

**Comment by zoltanszogyenyi** [comment]
Hello @RealBrianH,

Thanks for the kind thoughts and for following up. Let me take a clear stance on your questions:

1. I'm guessing you're referring to TailScan when you're talking about "promoting" other tools similar to DivMagic. One of the main reasons we do this is because they also published an article where they included Flowbite as one of the open-source UI libraries based on Tailwind CSS so it was a win-win situation. We have no issues with promoting this type of tool because **they're not promoting "copying content" from content that should not be copied from**.

2. Again, the issue at hand is not the tool or the capabilities of the tool itself, but the promotion of copying content from inside of iFrames that are not supposed to be copied from. It say "unlock the code" and that goes to our pricing page. Instead of that, the video you're showing bypasses that capability by showing that they don't necessarily need to. Does that make sense?

For example, if you would have shown that DivMagic is capable of copying content from our open-source sources such as from the tables that are present here: https://flowbite.com/docs/components/tables/ - we would have had no issue with you promoting this kind of feature. It's still an iFrame, right? :)

Please do follow up when that section has been removed.

Thanks,
Zoltan

**Comment by RealBrianH** [comment]
Hi @zoltanszogyenyi,

We've updated our website, and the section in question has been removed to address your concerns.

I was actually referring to Windy in our previous conversation. I understand the nuances and the win-win collaborations you've had with other tools.

Our aim is to provide value to developers, much like the tools you currently endorse. In fact, our tool can provide many of the functionalities offered by those listed tools combined as a single tool. Would it be possible for us to be included in your resources list as well? 

I look forward to your response.

Best,
Brian
